### PR TITLE
[Messenger] AsMessageHandler can be used on class methods

### LINKED ISSUE #16657: [Messenger] Allow AsMessageHandler attribute on methods
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#45273
| PR author(s) | @mjpvandenberg, @fabpot
| Merged in    | 6.1

### PR DESCRIPTION
**Pull request by mjpvandenberg**
Documents https://github.com/symfony/symfony/pull/45273

### CODE REVIEW

File: messenger.rst [hunk 1]
```diff
@@ -76,6 +76,13 @@ message class (or a message interface)::
     using PHP 7.4), by implementing :class:`Symfony\\Component\\Messenger\\Handler\\MessageHandlerInterface`
     instead.
 
+.. note::
+
+    As of Symfony 6.1, the ``AsMessageHandler`` attribute can be used on individual 
+    class methods as well. You may use the attribute on as many methods in a single
+    class as you like, allowing you to group the handling of multiple related types 
+    of messages.
```
Comment (wouterj) [inline messenger.rst:84]:
```suggestion
.. tip::

    You can also use the ``#[AsMessageHandler]`` attribute on individual class methods.
    You may use the attribute on as many methods in a single class as you like, allowing
    you to group the handling of multiple related types of messages.
    
.. versionadded:: 6.1

    Support for ``#[AsMessageHandler]`` on methods was introduced in Symfony 6.1.
```

Looks big (due to line breaks), but a minor suggestion: We always split describing a feature and saying "it's new". This way, we can blindly remove all 6.1 `versionadded` blocks in the future.

**Review by wouterj — APPROVED** [review]
Thank you for the quick reaction!

File: messenger.rst (hunk 1 above)
Comment (javiereguiluz) [reply messenger.rst:84]: Thanks Wouter. I did this change while merging.

**Comment by javiereguiluz** [comment]
Mark, thanks a lot for improving the docs with this missing feature ... and congrats on your first Symfony Docs contribution 🎉

**Pull request merged**